### DISCUSSION TITLE
Profitability stats for Rplant/OBTC

### DISCUSSION THREAD

**Discussion by smyers63** [RainbowMiner/RainbowMiner#2318] [Q&A]
What source has RBM used for OBTC profitability stats?  I've asked whattomine to add them but don't have a response yet.  The sources I know of don't list it.

**Comment by RainbowMiner** [comment]
The fallback solution is minerstat: https://minerstat.com/coin/OBTC

**Comment by smyers63** [comment]
I think I had seen that Minerstat listed OBTC but I wasn't aware of any mining software using Minerstat for getting the info.  Is the info from MInerstat obtained the same way it's obtained from whattomine when a mining controller like RBM requests the data?
